### ISSUE TITLE
Support multiple tags

### ISSUE THREAD

**Issue by JonathanDagan** [Shelex/cypress-allure-plugin#78]
**Is your feature request related to a problem? Please describe.**
Currently when using the ```.tag()``` function the test can only posses one tag. I would like the plugin to have the option to tag the test with multiple tags so when running the tests by their tag I could run the same test under two different tags.  

**Describe the solution you'd like**
I see two different ways to approach this issue. I would like to implement one of two options:

1. Add a ```.tags()``` function that receives a list of arguments -> ```.tags(['TAG1', 'TAG2'])```
1. Chain the ```.tag()``` function multiple times -> ```.tag('TAG1').tag('TAG2')```

The implementation needs to support typescript

**Additional context**
I would probably go for implementing the second option

**Comment by JonathanDagan** [comment]
Im currently working on trying to implement this

**Comment by Shelex** [comment]
Hi @JonathanDagan .
Thank you for opening a feature request. 

Actually I would go with extending signature for `.tag()` to accept string | array, rather than providing new command, as it makes a precedent to implement additional commands for other entities.

**Comment by Shelex** [comment]
@JonathanDagan Hi. Do you need any help with the issue? 
You can contact me by e-mail mentioned in my profile. In case you are not going to implement it - let me know anyway.

**Comment by JonathanDagan** [comment]
I will... Just was busy recently and will get to it asap 😄

**Comment by Shelex** [comment]
Hi @JonathanDagan .
Is there any progress on this feature?
Should I just implement it?)

**Comment by Shelex** [comment]
published 2.14.0, now tag function is variadic, so it can have multiple tags in same command with complete backward compatibility:
```js
cy.allure().tag('one')
cy.allure().tag('two', 'three', 'four')
```
